src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Listing/Listing.ts

```diff
@@ -147,7 +147,10 @@ export class Listing<Container extends ResourcesBase.Resource> {
                     return adhHttp.get($scope.path, params).then((container) => {
                         $scope.container = container;
                         $scope.poolPath = _self.containerAdapter.poolPath($scope.container);
-                        $scope.elements = _self.containerAdapter.elemRefs($scope.container);
+                        // FIXME: Sorting direction should be implemented in backend, working on a copy is used,
+                        // because otherwise sometimes the already reversed sorted list (from cache) would be 
+                        // reversed again
+                        $scope.elements = angular.copy(_self.containerAdapter.elemRefs($scope.container));
```

Comment:
IMHO angular should not be used directly (because it is not available in unit tests). Either use dependency injection or lodash.